library/node

```diff
@@ -68,10 +63,10 @@ Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
 GitCommit: bf40ad9d1c903c9d0d7ad71f0c1d7aea92f2051a
 Directory: 13/buster-slim
 
-Tags: 12.13.0-alpine, 12.13-alpine, 12-alpine, erbium-alpine, lts-alpine, current-alpine
-Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 93c5e098567620200e5a374622a86b4aff153506
-Directory: 12/alpine
+Tags: 13.1.0-alpine3.10, 13.1-alpine3.10, 13-alpine3.10, alpine3.10
+Architectures: 
```

Comment:
Travis is failing because of these empty `Architectures` lists

Reply:
Opened a PR at https://github.com/nodejs/docker-node/pull/1153